source/includes/_embed2.md.erb

```diff
@@ -442,6 +455,27 @@ Removes Embed2 from your page.
 
 Can be used to programatically open / close the Chat window. This method cannot be used with the `parentElement` option.
 
+#### triggerCampaign
+`triggerCampaign(campaignKey: string, options: object): Promise<void>;`
+
+Used to trigger the proactive Campaign with key `campaignKey`.
+
+An optional second argument `options` can be passed. These options may be helpful when testing your Campaign. The options are:
+`ignoreStatus?: boolean`: if `ignoreStatus` is `true`, the Campaign will be triggered even if it is inactive or in a draft state.
+`ignoreFrequency?: boolean`: if `ignoreFrequency` is `true`, the Campaign will be triggered even if it has already been triggered within the frequency configured on the Campaign.
+
+This action requires the Ada Engage Campaigns feature.
+
+#### trackEvent
+`trackEvent(trackEventParams: object)`
```

Comment:
I don't believe this is correct. I think the correct signature is `trackEvent(eventKey, value?, meta?)`

Reply:
As in, 3 parameters, not one

Reply:
Ooh you're right, the internal function takes it as an object but the external-facing one takes them separately - I'll update it!